src/engine/Resources/Sound.ts

```diff
@@ -582,6 +607,25 @@ class WebAudioInstance implements IAudio {
 
       return this._playingPromise;
    }
+   
+   public playWithProfile(){
+     if (this._isPaused) {
+       this._resume();
+     } else if (!this._isPlaying) {
+       this._start();
+     }
+     var point1 = 0.8;
+     var point2 = 0.1;
+     var point3 = 0.1;
+     var point4 = 0.4;
+     
+     while (new Date().getTime() < this._startTime + this._buffer.duration*1000) {
+       var cpt = (new Date().getTime() - this._startTime) / (this._buffer.duration*1000) ;
+       this.setVolume(((1-cpt)**3)*point1 + 3*((1-cpt)**2)*cpt*point2 + 3*(1-cpt)*(cpt**2)*point3 + (cpt**3)*point4);
```

Comment:
Also the TypeScript linter will fail on the math operators not being separated by a space.